pom.xml

```diff
@@ -129,10 +129,10 @@
   </ciManagement>
 
   <properties>
-    <license.title>Sonar C++ Plugin (Community) </license.title>
+    <license.title>Sonar C++ Plugin (Community)</license.title>
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
-    <sonar.version>4.5.2</sonar.version>
-    <sslr.version>1.20</sslr.version>
+    <sonar.version>5.2</sonar.version>
```

Comment:
Yeah, this is more to give an idea what we will be facing when moving to next lts. So far loads of changes. Anyway we should not merge this until lts is available